CONTRIBUTING.md

```diff
@@ -2,25 +2,25 @@
+Developed by [Getty](http://www.getty.edu), [Quire](https://quire.getty.edu) is a digital publishing tool that can create dynamic publications in a variety of formats, including web, print, and e-book. Quire is an ideal tool for publishing beautiful, scholarly digital books optimized for visual imagery and designed to ensure publication content is widely accessible and stable.
 
-Quire is open source and free to use. View Quire's [3-clause BSD open source license](https://github.com/thegetty/quire/blob/main/LICENSE). Please note: Quire v1 is currently in a pre-release phase. Testing and improvements are ongoing.
+Quire is open-source and free to use. View Quire's [3-clause BSD open source license](https://github.com/thegetty/quire/blob/main/LICENSE). Please note: Quire v1 is currently in a pre-release phase. Testing and improvements are ongoing.
 
-There are [many ways to contribute](#identify-a-contribution-to-make) including fixing bugs, improving the documentation, and creating a new feature or theme. Please read the following guidelines carefully. Our goal is to make the process as effective and transparent as possible, and to ensure that your contributions can become part of Quire. Thank you for taking the time.
+At the moment, the primary ways to contribute to Quire are reporting and fixing bugs or writing and improving the documentation. In time we will expand this to include suggesting features and enhancements. Please read the following guidelines carefully. Our goal is to make the process as effective and transparent as possible and to ensure that your contributions become part of Quire. Thank you for taking the time.
 
-🗣**CALLING ALL FIRST-TIMERS!** — We can’t stress enough that Quire is open to contributors at all levels. That could mean making a modest fix, like correcting a typo or fixing a broken link, or more intensive contributions like adding Quire CSS styles and JavaScript, or even copyediting one of our documentation pages. Here are a few resources especially for you:
+🗣**CALLING ALL FIRST-TIMERS!** — We can’t stress enough that Quire is open to contributors at all levels. That could mean making a modest fix, like correcting a typo in the docs, to a more intensive contribution, like fixing broken code. Here are a few resources especially for you:
 
-- Find [“good first issues”](https://github.com/thegetty/quire/issues) tagged in our issue tracker for the main thegetty/quire repo
-- Find [“good first issues”](https://github.com/thegetty/quire-docs/issues) tagged in our issue tracker for the thegetty/quire-docs repo
+- Find [“good first issues”](https://github.com/thegetty/quire/issues) tagged in our issue tracker for the main `thegetty/quire` repo

```

Comment:
```suggestion
- Find [“good first issues”](https://github.com/thegetty/quire/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) tagged in our issue tracker for the main `thegetty/quire` repo

```

This could link directly to the issues filtered by that label, though there are none yet